alf/examples/actor_critic.py

```diff
@@ -153,49 +157,54 @@ def create_algorithm(env,
 @gin.configurable
 def create_environment(env_name='CartPole-v0',
```

Comment:
The default env name could use 'SocialBot-CartPole-v0' here.

Reply:
I think it's better to use CartPole-v0, as it's more commonly available.